ACE/tests/SString_Test.cpp

```diff
@@ -150,6 +150,18 @@ int testConstIterator()
   return 0;
 }
 
+int testAutoStringFree()
+{
+  char* s = ACE_OS::strdup("Hello, World");
+  const ACE_Auto_String_Free s1(s);
+  if (!s1)
+    ACE_ERROR_RETURN ((LM_ERROR,
+                       ACE_TEXT ("ACE_Auto_String_Free is nullptr")),
+                       1);
+
```

Comment:
Also test when it is nullptr

Reply:
OK

Reply:
Test with a ACE_Auto_String_Free which is empty

Reply:
OK